client/components/domains/contact-details-form-fields/style.scss

```diff
@@ -86,6 +86,10 @@
 		min-width: 100px;
 	}
 
+	.phone-input__country-select {
+		min-width: 0;
```

Comment:
Huh -- TIL that Safari and IE don't support `min-width: auto;`. :+1: for the use of 0 here!